app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -0,0 +1,189 @@
+package org.jellyfin.androidtv.util.profile
+
+import org.jellyfin.androidtv.constant.CodecTypes
+import org.jellyfin.androidtv.constant.ContainerTypes
+import org.jellyfin.androidtv.util.DeviceUtils
+import org.jellyfin.androidtv.util.Utils
+import org.jellyfin.androidtv.util.profile.ProfileHelper.deviceHevcCodecProfile
+import org.jellyfin.androidtv.util.profile.ProfileHelper.subtitleProfile
+import org.jellyfin.apiclient.model.dlna.CodecProfile
+import org.jellyfin.apiclient.model.dlna.CodecType
+import org.jellyfin.apiclient.model.dlna.DirectPlayProfile
+import org.jellyfin.apiclient.model.dlna.DlnaProfileType
+import org.jellyfin.apiclient.model.dlna.ProfileCondition
+import org.jellyfin.apiclient.model.dlna.ProfileConditionType
+import org.jellyfin.apiclient.model.dlna.ProfileConditionValue
+import org.jellyfin.apiclient.model.dlna.SubtitleDeliveryMethod
+
+class ExoPlayerProfile(
+	isLiveTV: Boolean = false,
+	isLiveTVDirectPlayEnabled: Boolean = false,
+	isDtsEnabled: Boolean = true
+) : BaseProfile() {
+	init {
+		name = "AndroidTV-ExoPlayer"
+
+		directPlayProfiles = listOfNotNull(
```

Comment:
I think using `buildList {}` might be more readable here. Could also split the different profiles to functions/vals to make it easier.